qutebrowser/config/configdata.yml

```diff
@@ -503,13 +503,10 @@ content.host_blocking.whitelist:
   - piwik.org
   type:
     name: List
-    valtype: String
+    valtype: UrlPattern
     none_ok: true
   desc: >-
-    List of domains that should always be loaded, despite being ad-blocked.
-
-    Domains may contain * and ? wildcards and are otherwise required to exactly
-    match the requested domain.
+    List of URL Patterns that should always be loaded, despite being ad-blocked.
```

Comment:
LGTM now, accpet that I still wonder if we should include a description of `URLPattern`s in the descriptions and if we should use "URL pattern" instead of "URL Pattern" if we are not using `UrlPattern`. But this is something that I would take care of in a separate PR once there is a decision. So while I invite everybody to comment on this here, I see no need to tweak this PR anymore in this regard.

Reply:
"URL Pattern" looks weird to me (it's more of a concept than a name). For `completion.web_history.exclude` we just say "A list of patterns". Seems fine to use "A list of URL patterns" everywhere.

The type of the setting is already [clickable](https://github.com/qutebrowser/qutebrowser/blob/master/doc/help/settings.asciidoc#completion.web_history.exclude) in the docs, and the `UrlPattern` type description there does link to the Google docs, so I'd avoid repeating that in the docs for the actual settings.